docs/loopback/access-control.md

```diff
@@ -0,0 +1,24 @@
+# Authentication Authorization and Permissions
+
+## Dynamic-Role-Resolver
```

Comment:
Did you gain any insights into the inner workings of this? how are those roles resolved? How does it affect performance?